src/index.js

```diff
@@ -8,6 +10,78 @@ import $ from 'jquery';
 import './css/base.scss';
 
 // An example of how you tell webpack to use an image (also need to link to it in the index.html)
-import './images/turing-logo.png'
+// import './images/turing-logo.png'
 
-console.log('This is the JavaScript entry file - your code begins here.');
+import Game from './Game.js';
+import Wheel from './Wheel.js';
+
+let game;
+
+$('.startGame').click((event) => {
+  event.preventDefault();
+  let player1 = $('#player1Input').val();
+  let player2 = $('#player2Input').val();
+  let player3 = $('#player3Input').val();
+  let wheel = new Wheel();
+  game = new Game(wheel.wheel);
+  // console.log(game);
+  game.createPlayers(player1, player2, player3);
+  updatePlayer1();
+  updatePlayer2();
+  updatePlayer3();
+  game.makeNewRound()
+  $('.puzzleInputArea').html(`${appendPuzzle()}`);
+  $('.puzzleCharacter').hide();
+  $('.symbol').show();
+  $('.category').text(game.round.puzzle.category);
+  $('.hint').text(game.round.puzzle.hint);
+})
+
+$('.spinBtn').click((event) => {
+  event.preventDefault();
+  $('.spinValue').text(game.round.turn.spinWheel());
+})
+    
+
+
+function updatePlayer1() {
+  $('.player1Name').text(game.players[0].name);
+  $('.player1Score').text(game.players[0].score);
+  $('.player1Bank').text(game.players[0].bank);
+}
+
+function updatePlayer2() {
+  $('.player2Name').text(game.players[1].name);
+  $('.player2Score').text(game.players[1].score);
+  $('.player2Bank').text(game.players[1].bank);
+}
+
+function updatePlayer3() {
+  $('.player3Name').text(game.players[2].name);
+  $('.player3Score').text(game.players[2].score);
+  $('.player3Bank').text(game.players[2].bank);
+}
```

Comment:
Looks like we could DRY up some of these player function updates with a single function that takes in an argument of which player to update. e.g. if your function took in a `playerIndex` that was either 1, 2, or 3, you could make your logic a bit more flexible/dynamic by passing that variable through instead:

```js
$(`.player${playerIndex}Name`).text(game.players[playerIndex].name);
```